### PR TITLE
Version 1.300 (minus final kerning)

### PR DESCRIPTION
**Pull request by dbenjaminmiller**
Per the FONTLOG:

- Add small caps ‘smcp’ for the regular text font.
- Add T and S with comma below for setting Romanian.
- Add superior* and inferior figures.
- Add alternate old-style figures, accessible via ‘ss11’.
- Fix misplaced accents on ‘i’.*
- Add ‘ffj’ ligature.*
- Fix thickness of connector part in division symbol.*
_*Already merged_

Now, just to add kerning for some of the Latin-Extended characters.

### CODE REVIEW

File: FONTLOG.txt [hunk 1]
```diff
@@ -34,6 +34,16 @@ http://github.com/alif-type/xits
...
 ChangeLog
 ----------
+[Insert Date etc.] <XITS> Version 1.300
+- Add small caps ’smcp’ for the regular text font.
+- Add T and S with comma below for setting Romanian.
+- Add superior and inferior figures.
+- Add alternate old-style figures, accessible via ‘ss11’.
+- Fix misplaced accents on ‘i’.
+- Add ’ffj’ ligature.
+- Fix thickness of connector part in division symbol.
```
Comment (khaledhosny) [inline FONTLOG.txt]: I thought I fixed that one already.

File: sources/smallcaps.fea [hunk 2]
```diff
@@ -0,0 +1,309 @@
+lookup c2scCapitalstoSmallCapitalslookup1 {
```
Comment (khaledhosny) [inline sources/smallcaps.fea]: Make this lookup block a feature block (`feature c2sc {`) and remove the feature blocks below.

File: sources/smallcaps.fea [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+    sub \uni01FD by \uni01FD.sc ;
+    sub \uni01FF by \uni01FF.sc ;
+    sub \uni203C by \uni203C.sc ;
+    sub \uni2047 by \uni2047.sc ;
+    sub \uni214B by \uni214B.sc ;
+    sub \scommaaccent by \scommaaccent.sc ;
+    sub \uni021B by \uni021B.sc ;
+} smcpLowercasetoSmallCapitalslookup0;
+
+feature c2sc {
```
Comment (khaledhosny) [inline sources/smallcaps.fea]: Removes these feature blocks. The languages and scripts are declared globally to ensure consistency. Inconsistencies in these can mean some lookups are not applied for some languages.

File: FONTLOG.txt (hunk 1 above)
Comment (dbenjaminmiller) [reply FONTLOG.txt]: You did but it wasn't in fontlog as it was since the last release. So I added it to the notes for the forthcoming one

**Comment by dbenjaminmiller** [comment]
Fixed the issue with `smallcaps.fea`. The FONTLOG didn't previously include the changes in the upcoming version, so I had added them — it still is not quite final, as it should include the forthcoming kerning addition (once you or I apply it) and the ultimate date etc. when the release is finalized.

**Comment by khaledhosny** [comment]
> The FONTLOG didn't previously include the changes in the upcoming version,

Yes, I write them once when I’m making the release by checking the git log since the previous release tag.

**Comment by dbenjaminmiller** [comment]
I can walk it back for now, then.

**Comment by khaledhosny** [comment]
I squashed your commits into one and pushed a few cleanups and fixes. Hopefully I didn’t break anything.

**Comment by dbenjaminmiller** [comment]
Looks good.

**Comment by dbenjaminmiller** [comment]
CI build failed. It worked locally, no? Double check that, although the changes seemed fine

**Comment by khaledhosny** [comment]
That is because I force-pushed before the build started.

**Pull request merged**